user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModel.kt

```diff
@@ -45,7 +47,10 @@ class UserAccountsViewModel(private val repository: UserAccountsRepository,
                     userAccountLoading.set(it.isLoading)
                     isCurrentlyFetching()
                 }
-                is Success -> updatePatient(it.data?.linkedPatient)
```

Comment:
what is the need of this change?

Reply:
this code is not removed. few lines are added or removed.so showing like that